soc/renesas/ra/ra6m5/Kconfig.soc

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA6M5
+	bool
+	select SOC_FAMILY_RENESAS_RA
+	help
+		Renesas RA6M5 series
```

Comment:
help text indent is 1x tab followed by 2x spaces

Reply:
I updated, and similar cases were also. Thanks